SmartDeviceLink/SDLSecondaryTransportManager.m

```diff
@@ -352,6 +357,7 @@ - (void)sdl_configureManager:(nullable NSArray<SDLSecondaryTransportTypeBox *> *
     // this will trigger audio / video streaming if they are allowed on primary transport
     [self sdl_handleTransportUpdateWithPrimaryAvailable:YES secondaryAvailable:NO];
 
+    self.isAppReady = true;
```

Comment:
Why is this here instead of in `appDidBecomeReady`? Also remember that in Obj-C, true / false is YES / NO.